tools/README.md

```diff
@@ -0,0 +1,25 @@
+# glTF Sample Model Tools
+## generateSampleModels.py
+Python script to generate glTF models from COLLADA [source models](../sourceModels) using [COLLADA2GLTF](https://github.com/KhronosGroup/COLLADA2GLTF).
+
+Place a built COLLADA2GLTF binary in the tools folder and execute the script.
```

Comment:
This clutters up the repo with files that shouldn't be submitted.  Perhaps the user could set an environment variable or something to indicate where they keep their copy of COLLADA2GLTF, so that when it gets updated, this script always gets the update.